### PR TITLE
feat: more rop claims

### PR DESCRIPTION
**Pull request by hperl**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

File: docs/oauth2-oidc/resource-owner-password-grant.mdx [hunk 1]
```diff
@@ -48,3 +48,43 @@ import Mermaid from "@site/src/theme/Mermaid";
     Client->>User: Use access token
 `} />
 ```
+
+## Token introspection and acces token claims
+
+The access token that is issued can be introspected using the OAuth 2.0 Token Introspection endpoint. The following claims
+returned from the introspection endpoint, and part of the access token if the config paramter `strategies.access_token` is set to
```
Comment (vinckr) [inline docs/oauth2-oidc/resource-owner-password-grant.mdx:55]:
```suggestion
returned from the introspection endpoint, and part of the access token if the config paramater `strategies.access_token` is set to
```

File: docs/oauth2-oidc/resource-owner-password-grant.mdx [hunk 2]
```diff
@@ -48,3 +48,43 @@ import Mermaid from "@site/src/theme/Mermaid";
     Client->>User: Use access token
 `} />
 ```
+
+## Token introspection and acces token claims
```
Comment (vinckr) [inline docs/oauth2-oidc/resource-owner-password-grant.mdx:52]:
```suggestion
## Token introspection and access token claims
```

File: docs/oauth2-oidc/resource-owner-password-grant.mdx (hunk 1 above)
Comment (vinckr) [inline docs/oauth2-oidc/resource-owner-password-grant.mdx:55]:
```suggestion
The access token that is issued can be introspected using the OAuth 2.0 Token Introspection endpoint. The following claims are
returned from the introspection endpoint, and part of the access token if the config paramter `strategies.access_token` is set to
```

**Review by vinckr — COMMENTED** [review]
generally would be better to use active voice instead of passive voice.

so instead of 

"The claims in the access token and introspection data can be further customized using a token hook"

"You can customize the claims in the access token and introspection data using a token hook"

Would you agree that the second version is easier to understand?

**Pull request merged**